dispatcher/utils.py

```diff
@@ -1,14 +1,25 @@
 import importlib
 from enum import Enum
-from typing import Callable, Optional
+from typing import Callable, Optional, Protocol, Type, Union, runtime_checkable
+
+
+@runtime_checkable
+class RunnableClass(Protocol):
+    def run(self, *args, **kwargs) -> None: ...
+
+
+MODULE_METHOD_DEL = ':'
```

Comment:
Would you agree to rename it with a more meaningful name? For example `MODULE_METHOD_DELIMITER` ?
I think "DEL" can be ambiguous.